fix: Resource not accessible by integration error (#25)
nearform/github-action-check-linked-issues: README.md

## Standard Usage

> Configure a workflow to run a job on these `pull_request` or  `pull_request_target` events:

Reviewer: ```suggestion
Configure a workflow to run a job on these `pull_request` or  `pull_request_target` events.
```

> If you enable `comments` feature we recommend to listen `pull_request_target` event. 

Reviewer: ```suggestion
If you enable the `comments` option (enabled by default) we recommend to listen on `pull_request_target` event.
```

> `pull_request_target` event has write permission to the target repository allowing external forks create comments.

Reviewer: ```suggestion
`pull_request_target` event has write permission to the target repository allowing external forks to create comments.
```

```yaml
on:
  pull_request_target:

>     types: [opened, edited, reopened, synchronize]

Reviewer: does pull_request_target have all the same types as pull_request?

Author: Yes we have the same types [reference](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target)
